source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,195 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v8.0 - Major Release](#release-v8-0-major-release)
 - [Release v7.11 - Feature Release](#release-v7-11-feature-release)
 - [Release v7.10 - Feature Release](#release-v7-10-feature-release)
 - [Release v7.9 - Feature Release](#release-v7-9-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 
+## Release v8.0 - [Major Release](https://docs.mattermost.com/upgrade/release-definitions.html#major-release)
+
+**Release Day: July 14, 2023**
+
+### Important Upgrade Notes
+
+ - Insights has been disabled for all new instances and for existing servers that upgrade to v8.0. See more details in [this forum post](https://forum.mattermost.com/t/proposal-to-revise-our-insights-feature-due-to-known-performance-issues/16212) on why Insights has been disabled.
+ - The Focalboard plugin is now disabled by default and can be enabled in the **System Console > Plugin settings**.
+ - The Channel Export and Apps plugins are now disabled by default.
+ - Apps Bar is now enabled by default for on-prem servers. ``ExperimentalSettings.EnableAppBar`` was also renamed to ``ExperimentalSettings.DisableAppBar``. See more details at:
+   - https://docs.mattermost.com/configure/experimental-configuration-settings.html#disable-app-bar 
+   - https://forum.mattermost.com/t/channel-header-plugin-changes/13551
+ - The Go module has been upgraded to v8.0. All packages are now under the new path ``github.com/mattermost-server/server/v8``. Clients will have to change their module path to the new structure. Very minimal change. There are no API level breaking changes that would require code changes.
```

Comment:
This can be removed. https://github.com/mattermost/docs/pull/6369/files#diff-eba248b2c1a66750343693c8d84e6909fc3c26f299880afcc7261837a0465323R28-R29 covers the go module changes.
```suggestion
```